src/integrationTest/java/uk/gov/hmcts/darts/audio/controller/AudioControllerAddAudioMetadataIntTest.java

```diff
@@ -33,8 +29,6 @@
 import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.multipart;
 import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
 
-@SpringBootTest
-@ActiveProfiles({"intTest", "h2db"})
```

Comment:
Author note:
many integration tests extend `IntegrationBase` which uses profiles `intTest`, `h2db`, `in-memory-caching`,  but then explicitly re-declare profiles excluding `in-memory-caching`.
Profiles are inherited, as such the test runs with the `in-memory-caching` anyway.
This change assumes that the original intention was NOT to exclude in memory caching.

Reply:
Yeah, theres many other places where annotations have been redeclared.